### PR TITLE
[JENKINS-40003] Add plugin description to POM.

### PR DESCRIPTION
**Pull request by varyvol**
[JENKINS-40003](https://issues.jenkins-ci.org/browse/JENKINS-40003)

@reviewbybees, specially @stephenc

### CODE REVIEW

**Comment by rsandell** [comment]
:bee:

File: pom.xml [hunk 1]
```diff
@@ -37,6 +37,7 @@
   <packaging>hpi</packaging>
 
   <name>SSH Credentials Plugin</name>
+  <description>Allows to store SSH credentials in Jenkins</description>
   <url>http://wiki.jenkins-ci.org/display/JENKINS/SSH+Credentials+Plugin</url>
```
Comment (varyvol) [inline pom.xml]: Changed.

**Comment by recena** [comment]
🐝

**Comment by varyvol** [comment]
@stephenc could you please review this?

File: pom.xml [hunk 2]
```diff
@@ -37,7 +37,8 @@
   <packaging>hpi</packaging>
 
   <name>SSH Credentials Plugin</name>
-  <url>http://wiki.jenkins-ci.org/display/JENKINS/SSH+Credentials+Plugin</url>
+  <description>Allows to store SSH credentials in Jenkins</description>
```
Comment (stephenc) [inline pom.xml]: bad grammar

Comment (varyvol) [inline pom.xml]: @stephenc could you be more specific? I cannot see where the bad grammar is. The formula is quite similar to the one in [ssh-slaves-plugin PR #36](https://github.com/jenkinsci/ssh-slaves-plugin/pull/36/files).

Comment (stephenc) [inline pom.xml]: `Allows storage of SSH credentials in Jenkins` would be more correct.

"Allows to store" just sounds wrong to the ear of any native English speaker and we [cannot tell you why](https://twitter.com/MattAndersonBBC/status/772002757222002688)

Comment (varyvol) [inline pom.xml]: Fair enough. Changed.

**Comment by recena** [comment]
🐝

**Comment by varyvol** [comment]
@reviewbybees done. 

I think this can be merged whenever you consider it appropriate @stephenc

**Pull request merged**